src/ontology/mondo-edit.obo

```diff
@@ -513752,6 +513752,31 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7099" xsd:anyURI
 
+[Term]
+id: MONDO:0100535
+name: clonal hematopoiesis
+def: "Cellular proliferation of hematopoietic cells where a substantial proportion of the cells is derived from a single hematopoietic stem cell lineage." [NCIT:C162188, PMID:34215849]
+xref: NCIT:C162188 {source="MONDO:equivalentTo"}
+is_a: MONDO:0060782 {source="NCIT:C162188"} ! premalignant hematological system disease
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7137" xsd:anyURI
+
+[Term]
+id: MONDO:0100536
+name: clonal hematopoiesis of indeterminate potential
+def: "A term that refers to the presence of somatic mutations in bone marrow or peripheral blood cells in individuals who may be cytopenic but do not have morphologic evidence of hematologic neoplasia. Its prevalence rises with age and is found in approximately 10% of individuals aged 70 to 80. It is associated with an increased risk of hematologic neoplasia. Mutations in the DNMT3A, TET2, or ASXL1 genes are usually identified. Approximately 10%-40% of individuals with age-related clonal hematopoiesis will progress to meet the diagnostic criteria for clonal hematopoiesis of indeterminate potential." [NCIT:C158547]
```

Comment:
Make sure that the term refers to a disease. Maybe start the definition with
"A precancerous condition characterized by the presence of...."

Reply:
fixed